core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -606,7 +549,7 @@ private object RTS {
               // At this point, all causes of interruption have been accumulated
               // in the fiber status and will be read during evaluation of this
               // action:
-              curIo = IO.terminate
+              curIo = IO.interrupt
```

Comment:
The comment can be revised since we're deleting causes of interruption and turning `terminationCauses` into a simple `interrupted: Boolean`.